src/udp_capture.cpp

```diff
@@ -718,6 +787,7 @@ BFstatus bfUdpCaptureCreate(BFudpcapture* obj,
 }
 BFstatus bfUdpCaptureDestroy(BFudpcapture obj) {
 	BF_ASSERT(obj, BF_STATUS_INVALID_HANDLE);
+	obj->close_logs();
```

Comment:
This could be called from the destructor. Even better would be to wrap the `mkdir/rm` in a separate class's constructor and destructor, which would remove the need for `close_logs()` completely.